src/angularComponent/ngcOmniboxComponent.js

```diff
@@ -0,0 +1,6 @@
+import NgcOmniboxController from './ngcOmniboxController.js';
```

Comment:
How should I import the template? I don't think that little `!text` trick is going to work without SystemJS.